layouts/partials/footer.html

```diff
@@ -37,6 +37,21 @@
             <div class="down-main">
                 <div class='img-div'>
                     <!-- <a class='facebook-a' href="" target="_blank"></a> -->
+                    <a class='wechat' href="javascript:void(0);">
+                        {{ "/static/images/footer/wechat.svg" | readFile | safeHTML }}
+                        <div class="hide-div">
+                            <p>Follow the official account</p>
+                            <img src="/images/footer/wechat_code.svg" alt="">
+                        </div>
+                    </a>
+                    <a class='wechat' href="javascript:void(0);">
+                        {{ "/static/images/footer/group.svg" | readFile | safeHTML }}
+                        <div class="hide-div">
+                            <p>Join the WeChat group</p>
+                            <img src="/images/footer/group_code.svg" alt="">
+                            <p>进群请备注 “公司 - 姓名”</p>
```

Comment:
This icon is no need, remove it please.

![image](https://user-images.githubusercontent.com/40452856/84476376-9ca55280-acc0-11ea-9150-d7ff08376462.png)